sound/soc/codecs/max98373.h

```diff
@@ -212,5 +212,13 @@ struct max98373_priv {
 	bool interleave_mode;
 	unsigned int ch_size;
 	bool tdm_mode;
+	/* variables to support soundwire */
+	struct sdw_slave *slave;
+	bool hw_init;
+	bool pm_init_once;
+	int slot;
+	int rx_mask;
```

Comment:
it's assigned from and to `unsigned int`, better make it the same type

Reply:
Agreed. Shall fix this.